src/Prometheus/PushGateway.php

```diff
@@ -49,9 +49,9 @@ public function pushAdd(CollectorRegistry $collectorRegistry, $job, $groupingKey
      * @param $job
      * @param $groupingKey
      */
-    public function delete($job, $groupingKey = null)
+    public function delete(CollectorRegistry $collectorRegistry, $job, $groupingKey = null)
```

Comment:
`$collectorRegistry` is left out on purpose here. It's not being used anyways. Modeled after the [Python implementation](https://github.com/prometheus/client_python/blob/4ba6c90ee481433123682d027725c18a8f311150/prometheus_client/exposition.py#L253) which also leaves out the registry when doing a delete.